android/src/com/mapswithme/maps/MwmActivity.java

```diff
@@ -422,8 +421,6 @@ protected void onSafeCreate(@Nullable Bundle savedInstanceState)
 
     if (savedInstanceState == null && RoutingController.get().hasSavedRoute())
       addTask(new Factory.RestoreRouteTask());
-
-    updateViewsInsets();
```

Comment:
Isn't the proper ix is to call updateViewsInsets _before_ block:
```
if (isConsumed || isFirstLaunch)
      return;
```

Why should we reset insets listeners on each onResume?

Reply:
Ideally it should be called in onCreate, but the mwmActivity only exposes onSafeCreate which in some case is called too late for the first event. I'll try to find a better way to fix this but for now it fixes the issue at least.

Reply:
@arnaudvergnet please do your best ) We prefer higher quality/fewer code fixes instead of faster ones.